clippy.json

```diff
@@ -53,6 +56,12 @@
     ],
     "meetup.com": "Awesome event, i'll be back...",
     "ogame.gameforge.com": "Fly safe!",
-    "gmail.com": "Do you need glasses?",
-    "mail.google.com": "Do you need glasses?"
+    "gmail.com": [
+        "Do you need glasses?",
+        "Maybe try Outlook? wink wink..."
+    ],
+    "mail.google.com": [
+        "Do you need glasses?",
+        "Maybe try Outlook? wink wink..."
+    ],
```

Comment:
This comma is extraneous and file is not JSON valid. Pls adjust and you can check validity of your JSON here https://jsonlint.com/

Reply:
It was copy pasting

Reply:
I did the necessary changes